test/tools/unified-spec-runner/runner.ts

```diff
@@ -57,7 +53,7 @@ export async function runUnifiedTest(
   // If test.skipReason is specified, the test runner MUST skip this
   // test and MAY use the string value to log a message.
   if (test.skipReason) {
-    console.warn(`Skipping test ${test.description}: ${test.skipReason}.`);
+    ctx.skipReason = test.skipReason;
```

Comment:
Does the printout work the same? Or is there logic elsewhere to handle it now?

Reply:
It prints out correctly after the test along with our other skipReasons, printing inside the test like this made it appear before the test title, a little confusing.